tests/phpunit/src/Commands/Ssh/SshKeyListCommandTest.php

```diff
@@ -38,7 +38,10 @@ public function testUpload(): void {
     $this->prophet->checkPredictions();
     $output = $this->getDisplay();
     $this->assertStringContainsString('Local filename', $output);
-    $this->assertStringContainsString('Cloud Platform key label', $output);
+    $this->assertStringContainsString('Label', $output);
+    $this->assertStringContainsString('UUID', $output);
+    $this->assertStringContainsString('sha256', $output);
```

Comment:
`hash` or no `hash`, that's the question.

Reply:
This is just a partial string match to ensure the label is there.